tests/input/credential-issuer-missing-fail.json

```diff
@@ -5,7 +5,7 @@
   "type": [
     "VerifiableCredential"
   ],
-  "issuer": {},
+  "issuer": null,
```

Comment:
technically I think this one should be `{issuer: null}`, but I don't think taken the complexity of this test suite not covering that particular fail state is the biggest deal especially with the rename so that is fine.